stdlib/public/runtime/SwiftRT-ELF-WASM.cpp

```diff
@@ -84,6 +84,7 @@ namespace {
 static swift::MetadataSections sections{};
 }
 
+SWIFT_ALLOWED_RUNTIME_GLOBAL_CTOR_BEGIN
```

Comment:
Can we `#include swift/Runtime/Config.h` explicitly? This looks like it's getting included transitively through `Backtrace.h`.

Thanks